dao/patient/etl-patient-dao.js

```diff
@@ -292,7 +292,7 @@ module.exports = function () {
 
         var queryParts = {
             columns: request.query.fields || "t1.*, t2.cur_arv_meds",
-            table: "etl.flat_labs_and_imaging",
+            table: "etl.flat_labs_and_imaging_onc_test",
```

Comment:
Any reason why we are using `flat_labs_and_imaging_onc_test` rather than the updated `flat_labs_and_imaging_onc`

Reply:
All lab data summary data(HIV, CDM and Oncology) is stored in flat_labs_and_imaging table. Onc test was used for testing purposes.

Reply:
Changed to etl.flat_labs_and_imaging